package.json

```diff
@@ -25,5 +26,6 @@
     "autoprefixer": "^10.4.19",
     "postcss": "^8.4.38",
     "tailwindcss": "^3.4.4"
-  }
+  },
+  "packageManager": "npm@10.9.1+sha512.c89530d37c4baa38afd43e76a077a84b9aa63840b986426584fd5c5a54ab0a0b21bb1595c851042b733784b0b43706d36a494b4d8ae1a086a762cb8d3f95942a"
```

Comment:
How about using pnpm?

Reply:
This repo is managed by npm. I will revert this line of changes.  We can submit a PR to change it to pnpm later.